ntc_templates/templates/alcatel_aos_show_system.textfsm

```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
+Value CONTACT (.+,(?: https://\S+)?)
+Value NAME ([^,]+)
+Value LOCATION ([^,]+)
+Value Service (\d+)
+Value DATE_TIME (\S+\s*\S+\s*\d+\s*\d+\s*\d+:\d+:\d+\s*(\(\w+\)))
```

Comment:
```suggestion
Value DATE_TIME ((\S+\s*)+)
```

Simplify regex.

Fortunately you can (and are) anchoring the match line with that trailing comma. :grinning: